vb-be/src/modules/user/user.controller.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Controller,
+  Get,
+  Response,
+  Headers,
+  Param,
+  Logger,
+} from '@nestjs/common';
+import { Response as Res } from 'express';
+
+import { UserService } from '@modules/user/user.service';
+import { IDecodedToken } from '@modules/auth/auth.constants';
+import { HttpStatus } from 'src/util/http';
```

Comment:
nit: `src` imports above `@modules` like we mentioned previously

Reply:
wait shouldn't the `@` module come first?